core/internal/runconfig/runconfig.go

```diff
@@ -84,7 +84,7 @@ func (runConfig *RunConfig) ApplyChangeRecord(
 		); err != nil {
 			onError(
 				fmt.Errorf(
-					"config: failed to unmarshall JSON for config key %v: %v",
+					"config: failed to unmarshal JSON for config key %v: %v",
```

Comment:
wow! i think i've always misspelled this

Reply:
yeah me too...  i just followed the spalling of the function